csp/tests/test_examples.py

```diff
@@ -0,0 +1,26 @@
+
+import importlib
+import os.path
+import pytest
+import sys
+
+EXAMPLES_ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "..", "examples"))
+
+# make examples importable without keeping in source tree
+sys.path.append(EXAMPLES_ROOT)
+
+def _get_module(folder, filename):
+    # Don't fail during test gathering
+    try:
+        return importlib.import_module(f"{folder}.{filename.replace('.py', '')}")
+    except BaseException:
```

Comment:
you should do `except Exception` since this will catch things like `SystemExit` and `KeyboardInterrupt`, which you probably don't want.